autopush/db.py

```diff
@@ -363,6 +379,7 @@ def fetch_notifications(self, uaid):
 
     @track_provisioned
     def save_notification(self, uaid, chid, version):
+        # type: (str, str, Optional[int]) -> bool
```

Comment:
delete's handling of that looks like a special case for unregistering a channel.. but I don't get why it would even call delete like that in that case